### PR TITLE
php updates and new packages: php8.*-redis-6.0.2

### PR DESCRIPTION
**Pull request by TinfoilSubmarine**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

**Comment by TinfoilSubmarine** [comment]
This one is also ready to merge, I added `php8.3-redis` to it.

**Comment by TinfoilSubmarine** [comment]
Rebased on latest PHP 8.3 changes and ready for merge again

File: srcpkgs/php8.1-redis/INSTALL.msg
Comment (classabbyamp) [inline srcpkgs/php8.1-redis/INSTALL.msg:1]: README.voidlinux not INSTALL.msg

File: srcpkgs/php8.2-redis/INSTALL.msg
Comment (classabbyamp) [inline srcpkgs/php8.2-redis/INSTALL.msg:1]: same

File: srcpkgs/php8.3-redis/INSTALL.msg
Comment (classabbyamp) [inline srcpkgs/php8.3-redis/INSTALL.msg:1]: same

**Comment by classabbyamp** [comment]
```
post_install() {
vdoc $FILESDIR/README.voidlinux
}
```
in each

**Comment by TinfoilSubmarine** [comment]
> post_install() {
> vdoc $FILESDIR/README.voidlinux
> }

:facepalm: my bad, fixed

**Pull request merged**